### ISSUE TITLE
Bug | Add scrollbar to timeline

### ISSUE THREAD

**Issue by febrezo** [cyb3rfox/Aurora-Incident-Response#39]
Just noticed that when the list of events exceeds the size of the window, it is not possible to scroll down the view. A scrollbar may fix this avoiding manual window resize.

**Comment by cyb3rfox** [comment]
Yes i noticed that. The problem is just, that the mousewheel is already attached to the size up and down operations. I'd rather limit the zoom factor so it stops once the screen is hit. But so far no idea how to do that.
